React-frontend/src/components/core/Navbar.js

```diff
@@ -1,60 +1,304 @@
+                color="textPrimary"
+                to="/members"
+                component={RouterLink}
+                className={classes.link}
+              >
+                Members
+              </Link>
+
+              <Link
+                variant="button"
+                color="textPrimary"
+                to="/contact-us"
+                component={RouterLink}
+                className={classes.link}
+              >
+                Contact us
+              </Link>
+
+              <Link
+                variant="button"
+                color="textPrimary"
+                to="/dashboard"
+                component={RouterLink}
+                className={classes.link}
+              >
+                Dashboard
+              </Link>
+            </>)
+            : (<>
+              <Link
+                variant="button"
+                color="textPrimary"
+                to="/"
+                component={RouterLink}
+                style={{ marginRight: '20px' }}
+                className={classes.link}
+              >
+                Home
+              </Link>
+
+              <Link
+                variant="button"
+                color="textPrimary"
+                to="/https://github.com/scorelab/OpenMF/"
+                component={RouterLink}
+                style={{ marginRight: '20px' }}
+                className={classes.link}
+              >
+                Help
+              </Link>
+            </>
+            )
+          )
+        }
+      </nav>
+
+      {/* Display login option or user menu as per authenticated status */}
+      {
+        (!auth.isAuthenticated ? (
+        <>
+          <Button
+            size="small"
+            component={RouterLink}
+            to="/login"
+            color="primary"
+            className={classes.button}
+            variant="outlined"
+          >
+            Login
+          </Button>
+
+          <Button
+            size="small"
+            component={RouterLink}
+            to="/register"
+            color="primary"
+            variant="outlined"
+            className={classes.button}
+          >
+            Register
+          </Button>
+        </>)
+          : (<>
+
+            <IconButton
+              edge="start"
+              style={{ marginLeft: 'auto' }}
+              color="primary"
+              aria-controls="simple-menu"
+              aria-haspopup="true"
+              onClick={handleClick}
+            >
+              <Avatar className={classes.purple}>
+                {(auth && auth.user) ? auth.user.name.charAt(0).toUpperCase(): (<span>wait...</span>)}
+              </Avatar>
+            </IconButton>
+            <Menu
+              id="simple-menu"
+              anchorEl={anchorEl}
+              keepMounted
+              open={Boolean(anchorEl)}
+              onClose={handleClose}
+              TransitionComponent={Fade}
+              style={{ marginTop: '25px' }}
```

Comment:
Here also!

Reply:
@swapnalshahil thanks for reviewing, but in some of the place, I used pixels instead of % as those margins need to be same for all kind of device frames :)

Reply:
Ok cool then 😀👍